Neos.Neos/Resources/Private/Templates/Module/Management/Workspaces/Show.html

```diff
@@ -21,7 +21,7 @@
 										<input type="checkbox" id="check-all" /><span></span>
 									</label>
 								</th>
-								<th></th>
+								<th>{neos:backend.translate(id: 'workspaces.checkbox.label', source: 'Modules', package: 'Neos.Neos')}</th>
```

Comment:
That is a very generic, err, label. ;) It's not as if there is only one checkbox in the workspace module. May I suggest this:

```suggestion
								<th>{neos:backend.translate(id: 'workspaces.selectAllCurrentChanges', source: 'Modules', package: 'Neos.Neos')}</th>
```